Include/Rover/Lift.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef ROVER_LIFT_HPP
```

Comment:
Do you know why ```Lift``` is appearing in this pull request?

Reply:
I don't know, it shouldn't. That commit is already in master but in another order. I'll rebase to the exact same base.